RoddyCore/src/de/dkfz/roddy/Roddy.java

```diff
@@ -217,7 +217,11 @@ public static void main(String[] args) {
             mainStarted = true;
             startup(args);
         } catch (Exception e) {
-            e.printStackTrace();
+            // I did not want to put in a dependency to another library. This special type of
+            // exception is thrown by groovyclient (groovyserver project). As it is not known
+            // to the JVM in this case, we'll filter it by its name.
+            if (!e.getClass().getName().endsWith("SystemExitException"))
+                e.printStackTrace();
```

Comment:
No, here I'll ignore the SystemExitException (see the negation). As I wrote in the comment, I did not want to add another library dependency.